modules/bridge/common/bridge_header.cc

```diff
@@ -60,12 +57,11 @@ bool BridgeHeader::Diserialize(const char *buf) {
       cursor += sizeof(size_t) + size + 2;
       i -= sizeof(HType) + sizeof(size_t) + size + 3;
       continue;
-    } else {
-      size_t value_size = 0;
-      for (int i = 0; i < Header_Tail; i++) {
-        if (type == header_item[i]->GetType()) {
-          cursor = header_item[i]->DiserializeItem(cursor, &value_size);
-        }
+    }
+    size_t value_size = 0;
+    for (int i = 0; i < Header_Tail; i++) {
+      if (type == header_item[i]->GetType()) {
+        cursor = header_item[i]->DiserializeItem(cursor, &value_size);
       }
       i -= value_size;
```

Comment:
I reverted this change. Because
1. It changes the logic. The `i -= value_size;` was moved from outside to inside of the for-loop.
2. It cannot compile.